doc/release-notes-16528.md

```diff
@@ -0,0 +1,108 @@
+Wallet
+------
+
+### Experimental Descriptor Wallets
+
+Please note that Descriptor Wallets are still experimental and not all expected functionality
+is available. Additionally there may be some bugs and current functions may change in the future.
+
+0.21 introduces a new type of wallet - Descriptor Wallets. Descriptor Wallets store
+scriptPubKey information using descriptors. This is in contrast to the Legacy Wallet
+structure where keys are used to generate scriptPubKeys and addresses. Because of this
+shift to being script based instead of key based, many of the confusing things that legacy
+wallets are not possible with Descriptor Wallets. Descriptor Wallets determine use a definition
```

Comment:
> many of the confusing things that legacy wallets are not possible with Descriptor Wallets

There is a word missing in that sentence. Not exactly sure which one, maybe "the confusing things that legacy wallets _does_"?

Reply:
Fixed.